test/e2e-go/cli/goal/expect/README.md

```diff
@@ -77,6 +77,15 @@ go test -v test/e2e-go/cli/goal/expect/goal_expect_test.go
 
 TESTFILTER=foobar[0-9]Test TESTDIR=baz go test -v test/e2e-go/cli/goal/expect/goal_expect_test.go
 ```
+In particular, if you want to run `goalAccountInfoTest.exp` you'll need to create a temporary directory such as `deletemeTemp` with
```

Comment:
Adding more explicit instructions for running `*.exp` tests

Reply:
interesting.. I was always using the following syntax:
```sh
cd test/e2e-go/cli/goal/expect
go test -v . --run TestGoalWithExpect/goalCmdFlagsTest.exp
```

Reply:
ohhh... I only saw this now. Will try to get this into the README next PR.